cmd/acmesolver/app/app.go

```diff
@@ -32,7 +32,7 @@ func NewACMESolverCommand(ctx context.Context) *cobra.Command {
 	)
 
 	cmd := &cobra.Command{
-		Use:   "acme-solver",
+		Use:   "cert-manager-acme-solver",
```

Comment:
`acmesolver` is the name of the binary - `cert-manager-acmesolver` is the name of the docker image